### ISSUE TITLE
Add the missing Accessibility resource

### ISSUE THREAD

**Issue by nesk** [nesk/puphpeteer#39]
I forgot to add [the `Accessibility` resource](https://pptr.dev/#?product=Puppeteer&version=v1.12.2&show=api-class-accessibility), this is an easy fix.
